hack/create-gke-cluster.sh

```diff
@@ -34,7 +34,7 @@ source "$ROOT"/common.sh
 # shellcheck disable=SC1090
 source "$ROOT"/functions.sh
 
-# enable googleapi for container registry on the project
+enable googleapi for container registry on the project
```

Comment:
Should this still be a comment?

Reply:
Nice catch. I thought I backed out those changes completely

Reply:
Fixed